Adapter.C

```diff
@@ -186,6 +194,14 @@ bool preciceAdapter::Adapter::configFileRead()
             }
         }
 
+        // If the Volume_Coupling module is enabled, create it, read the
+        // Volume_Coupling-specific options and configure it.
+        if (Volume_Couplingenabled_)
```

Comment:
(long term / document)

With this approach, we have the choice between `{CHT, FSI, FF, Volume Coupling}`. In the long run, we want volume coupling to be possible for all (?) of `{CHT, FSI, FF}`.